[EEG] Add view/edit functionality and permissions for EEG BIDS derivatives (#7433)
aces/Loris: modules/electrophysiology_browser/README.md

## Purpose

The Electrophysiology Browser is intended to allow users to view candidate
electrophysiology (EEG, MEG...) sessions collected for a study and any associated

> annotations or derivative datasets for the session.

Reviewer: ```suggestion
annotations (derivatives) for each recording.
```
updating for accuracy

The Electrophysiology Browser displays electrophysiology datasets that have been
inserted into LORIS from a BIDS-format collection. Derived or processed electrophysiology

> datasets can also be accessed and edited via this module.

Reviewer: ```suggestion
datasets can also be accessed and annotated via this module.
```
annotated not edited.

## Permissions

The Electrophysiology Browser uses the following permissions. Either of the first two is 
sufficient to provide access to view data in the module. The third permission provides editing

> permissions for derivative datasets.

Reviewer: ```suggestion
permissions to add or modify annotations (derivatives) for data from all sites in this module.
```
@AlexandraLivadas  can you update these words `from all sites` if not accurate ?  does your permission to add/edit on all-sites or just the user's affiliated sites?

Author: I will update this because it does not apply for all-sites. The 2 permissions that existed before determine if the session can be viewed and this permission just determines if the annotations of the session being viewed can be edited.

electrophysiology_browser_view_allsites
  - This permission gives the user access to all electrophysiology datasets present in the database.
  
electrophysiology_browser_view_site
  - This permission gives the user access to electrophysiology datasets from their own site(s) only.
  

> electrophysiology_browser_edit_annotations

Reviewer: perfect. 👍

  - This permission allows the user to add, edit, and delete annotations for raw or derived datasets

## Download

> You can download all the files related to a recording and its derivatives (channel information,
> electrode information, task event information, the actual recording, the annotations and their metadata...).

Reviewer: ```suggestion
You can download all the files related to a recording (channel information,
electrode information, task event information, the actual recording) -- as well as its derivatives (annotations and their metadata).
```
updated this for clarity.  also no ellipses in documentation if you can help it...

## Updating Derivative Files

> When annotations are added or edited in the browser, updates are maded in the database but not directly in the associated derivative files for this data. These derivative files must also be updated. When a download is slected in the browser, the derivative files for the selected dataset are updated before downloading is triggered. All derivative files in the file system can be updated using the [update_annotation_files](../../tools/update_annotation_files.php) tool provided. This tool can be triggered to run automatically using a cronjob: 

Reviewer: proofreading / please correct: 
maded 
directly -> immediately
slected -> selected
downloading is triggered -> the actual download process starts
to simplify, Replace "All derivative files in the file system can be updated using the..." with "A script is provided to perform this [script name and link."

> This commmand will run the script on the first day of every month at midnight.   

Reviewer: mmm

Reviewer: ```suggestion
Below is a sample crontab entry to execute this script *every night* at midnight  
```
Please also change the cronjob fields so that it's not first of the month, but every night
